.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ruby.specSaveFile": true,
-  "ruby.specCommand": "HEADFULL=true RAILS_ENV=test bundle exec rspec",
+  "ruby.specCommand": "HEADFULL=true RAILS_ENV=test rspec",
```

Comment:
I don't know that this file should be in the repo.

Reply:
Yes. You're right, it shouldn't be commited to the repo. I'll remove it.